monkey/monkey_island/cc/next_ui/src/providers/authProvider/provider.tsx

```diff
@@ -25,7 +24,8 @@ export function AuthProvider({ children }: { children: ReactNode }) {
         const currentPathName: string = window.location.pathname;
         isSessionHasError(data);
```

Comment:
What is this doing?

Reply:
Checking for the error property's value.

Reply:
It's already being checked in the following if statement?